client/extensions/woocommerce/state/action-types.js

```diff
@@ -1,3 +1,5 @@
+export const WOOCOMMERCE_API_ERROR = 'WOOCOMMERCE_API_ERROR';
```

Comment:
I actually thought about changing this to `WOOCOMMERCE_SET_API_ERROR` to correspond with CLEAR better. Would that work?